docs/index.md

```diff
@@ -239,10 +239,18 @@ resource "databricks_user" "my-user" {
 resides. Alternatively, you can provide this value as an environment variable `DATABRICKS_AZURE_TENANT_ID` or `ARM_TENANT_ID`.
 * `azure_environment` - (optional) This is the Azure Environment which defaults to the `public` cloud. Other options are `german`, `china` and `usgovernment`. Alternatively, you can provide this value as an environment variable `ARM_ENVIRONMENT`.
 * `pat_token_duration_seconds` - The current implementation of the azure auth via sp requires the provider to create a temporary personal access token within Databricks. The current AAD implementation does not cover all the APIs for Authentication. This field determines the duration in which that temporary PAT token is alive. It is measured in seconds and will default to `3600` seconds. 
+
+There are multiple environment variable options, the `DATABRICKS_AZURE_*` environment variables take precedence, and the `ARM_*` environment variables provide a way to share authentication configuration using the `databricks` provider alongside the `azurerm` provider.
+
+## Miscellaneous configuration parameters
+
+This section covers configuration parameters not related to authentication.  They could be used when debugging problems, or do an additional tuning of provider's behaviour:
+
+* `rate_limit` - defines maximum number of requests per second made to Databricks REST API by Terraform. Default is *15*.
```

Comment:
This is the case where I'm not sure we should surface `rate_limit` and `skip_verify` in public documentation. @mgyucht @pietern for second opinion on this :)

Reply:
It's an advanced knob though, so it should be marked as such, like the others in this section.